src/_pytest/capture.py

```diff
@@ -102,6 +102,9 @@ def _getcapture(self, method):
 
     # Global capturing control
 
+    def is_globally_capturing(self):
+        return self._method != "no"
+
```

Comment:
I've thought about having a `is_suspended` property instead also.
What do you think?

Reply:
That would be `self._method != "no" and not self._suspended`, which is set internally then - or just `self.suspended`, which never gets set for method "no".

Reply:
I think this is fine actually.